tissue-clicker/src/App.js

```diff
@@ -7,24 +8,42 @@ import About from './About';
 import { ClickProvider } from './ClickContext';
 
 function App() {
+  const [activeMenu, setActiveMenu] = useState('');
```

Comment:
functions written to cater to the toggling of the menus instead of navigating to the separate 'pages'